test/test_surface_fluxes.jl

```diff
@@ -86,8 +86,8 @@ _fractional_indices(at_node, grid, ::Nothing, ::Nothing, ::Nothing) = (nothing,
                 set!(ocean.model, u = uₐ, v = vₐ, T = Tₒ)
 
                 # Compute the turbulent fluxes (neglecting radiation)
-                coupled_model    = OceanSeaIceModel(ocean; atmosphere, similarity_theory)
-                turbulent_fluxes = coupled_model.fluxes.turbulent.fields
+                coupled_model    = OceanSeaIceModel(ocean; atmosphere, ocean_similarity_theory=similarity_theory)
+                turbulent_fluxes = coupled_model.fluxes.turbulent.ocean.fields
```

Comment:
Just thinking about this... would we rather write `fluxes.turbulent.ocean` or `fluxes.ocean.turbulent`?

I'm ok to keep this pattern but wanted to have the discussion